nima/http/media/multipart/src/main/java/io/helidon/nima/http/media/multipart/MultiPartSupport.java

```diff
@@ -45,8 +47,13 @@ public class MultiPartSupport implements MediaSupport {
             .withParameter("boundary", DEFAULT_BOUNDARY);
 
     private MediaContext context;
+    private final ParserMode parserMode;
 
-    private MultiPartSupport() {
+    private MultiPartSupport(Config config) {
+        ConfigValue<String> parserModeConfig = config.get("parser-mode").as(String.class);
```

Comment:
where was the test for using config?

Reply:
This was reverted. No more tests are needed. :)